src/components/addView/addView.scss

```diff
@@ -0,0 +1,126 @@
+select {
+  display: block;
+  width: 100%;
+  max-width: 60ch;
+  padding: 0.85rem 1rem;
+  border: 1px solid $border-pale-grey;
+  border: 1px solid;
```

Comment:
Duplication here :slightly_smiling_face:

Reply:
@faysvas, thanks. Added the changes